DataStructure/src/main/list/ResizableArray.java

```diff
@@ -0,0 +1,164 @@
+package main.list;
+
+/**
+ * Creates a Resizable List of Integer elements like the ArrayList Implementation in Java
+ */
+
+public class ResizableArray {
+    private int size;
+    private int indexOfLastElement;
+    private int list[];
+
+    /**
+     * @param size
+     * Takes in the size of the list.
+     * If left blank default size of 50 will be used
+     */
+    public ResizableArray(int size){
+        this.size = size;
+        list = new int[this.size] ;
+        indexOfLastElement = 0;
+    }
+
+    public ResizableArray(){
+        size = 50;
+        list = new int[50] ;
+        indexOfLastElement = 0;
+    }
+
+    /**
+     * @param num
+     * Takes the number to add at the end of the list.
+     */
+    public void add(int num){
+        if (indexOfLastElement == size-1){
+            size = size + 50;
+            int list2 [] = new int[size];
+            for (int i = 0; i <= indexOfLastElement; i++) {
+               list2[i] = list[i];
+            }
+            list = list2;
+        }
+        list[indexOfLastElement] = num;
+        indexOfLastElement++;
+    }
+
+    /**
+     *@param index
+     * Gets the element at index
+     * @return
+     * Returns the element
+     */
```

Comment:
method description - get element at the given index.
param description - index of the element

Reply:
You should throw illegal argument exception when index is outside bounds.